src/pages/admin.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { withCustomUrqlClient } from "../components/CustomUrqlClient";
+import { Role } from "components/Role";
+import { Users } from "components/Users";
```

Comment:
is `../components` `components` ?

```suggestion
import { withCustomUrqlClient } from "components/CustomUrqlClient";
import { Role } from "components/Role";
import { Users } from "components/Users";
```